lib/reply.js

```diff
@@ -198,11 +198,12 @@ Reply.prototype.headers = function (headers) {
 }
 
 Reply.prototype.code = function (code) {
-  if (statusCodes[code] === undefined) {
+  const intValue = parseInt(code)
+  if (isNaN(intValue) || intValue < 100 || intValue > 600) {
     throw new FST_ERR_BAD_STATUS_CODE()
```

Comment:
We might want to pass the invalid `code` inside the error message.

Reply:
Fixed